exasol_script_languages_developer_sandbox/lib/run_install_dependencies.py

```diff
@@ -0,0 +1,30 @@
+from typing import Tuple
+
+from exasol_script_languages_developer_sandbox.lib.ansible.ansible_access import AnsibleAccess
+from exasol_script_languages_developer_sandbox.lib.ansible.ansible_context_manager import AnsibleContextManager
+from exasol_script_languages_developer_sandbox.lib.ansible.ansible_repository import AnsibleRepository, \
+    default_repositories
+from exasol_script_languages_developer_sandbox.lib.ansible.ansible_run_context import AnsibleRunContext, \
+    default_ansible_run_context
+
+from exasol_script_languages_developer_sandbox.lib.host_info import HostInfo
+
+
+def run_install_dependencies(ansible_access: AnsibleAccess, host_infos: Tuple[HostInfo, ...],
+                             ansible_run_context: AnsibleRunContext = default_ansible_run_context,
+                             ansible_repositories: Tuple[AnsibleRepository, ...] = default_repositories) -> None:
+    """
+    Runs ansible installation. The ansible working dir is created dynamically and removed afterwards.
+    Any hosts, given by variable host_infos, are added to the ansible inventory in the dynamic working directory.
+    All ansible repositories, given by variable ansible_repositories,
+    are copied as flat copy to the dynamic working copy, too.
+    The playbook is indicated by variable ansible_run_context, which also might contain additional ansible variables.
+    """
+    # TODO: Awaiting release 5.0.0 of slc!!!
```

Comment:
Create a ticket for the Todo

Reply:
Done https://github.com/exasol/script-languages-developer-sandbox/issues/12